### PR TITLE
bets + redeem

### PR DESCRIPTION
**Pull request by CleanBread**

### CODE REVIEW

File: examples/nextjs/src/app/bets/page.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import { useBets, OrderDirection } from '@azuro-org/sdk'
+import { useAccount, Address } from 'wagmi';
+import { BetCard } from '@/components';
+
+const useData = () => {
+  const { address } = useAccount()
+
+  const props = {
+    filter: {
+      bettor: address as Address,
```
Comment (on47sky) [inline examples/nextjs/src/app/bets/page.tsx]: it should be already Address, isn't it?

File: examples/nextjs/src/components/BetCard.tsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                </div>
+                <p>{GameStatusText[getGameStatus({ graphGameStatus: gameStatus, startDate: startsAt * 1000 })]}</p>
+              </div>
+              <div className="flex items-center">
+                <Link href={`/events/${sportSlug}/${gameId}`} className="flex items-center mr-4">
+                    {
+                      participants.map(({ image, name }) => (
+                        <div key={name} className="flex items-center ml-2 first-of-type:ml-0">
+                          <div className="flex items-center justify-center w-8 h-8 mr-2 border border-zinc-300 rounded-full">
+                            <img className="w-4 h-4" src={image || undefined} alt="" />
```
Comment (on47sky) [inline examples/nextjs/src/components/BetCard.tsx]: maybe it's better to wrap?
```
Boolean(image) && (
  <img />
)
```

**Review by on47sky — APPROVED** [review]

File: examples/nextjs/src/components/BetCard.tsx (hunk 2 above)
Comment (CleanBread) [reply examples/nextjs/src/components/BetCard.tsx]: 4f270bf8149281eb738cbec9d0042e37ea99ea86

File: examples/nextjs/src/app/bets/page.tsx (hunk 1 above)
Comment (CleanBread) [reply examples/nextjs/src/app/bets/page.tsx]: 03bd6c2a2a70f0f46610776ea9f89690af92900f

**Pull request merged**